build_projects/dotnet-host-build/PrepareTargets.cs

```diff
@@ -244,6 +273,11 @@ public static BuildTargetResult RestorePackages(BuildTargetContext c)
                 .Execute()
                 .EnsureSuccessful();
 
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel")
+                .WorkingDirectory(Path.Combine(c.BuildContext.BuildDirectory, "tools", "dotnet-deb-tool"))
```

Comment:
I don't think this will work since one of the calls utilizes a global.json inside the `independent` folder but I'll try it out. 

:+1: for `RepoRoot` Been wanting to get rid of `BuildDirectory` everywhere for some time